test/reporter-stdout.js

```diff
@@ -0,0 +1,21 @@
+/*globals QUnit:true*/
+/*jshint node:true*/
+var QUnit = require( "../dist/qunit" );
+
+// Options: { output: "minimal" || "verbose" }
+QUnit.stdout();
+
+// Load QUnit tests
+require( "./logs" );
```

Comment:
Looks like the `this` from outro.js refers in node to `global`, which we then alias as `window`, and in exports.js we then assign `window.QUnit`, which in node is the same as `global.QUnit`. Let's fix that.